src/styles/site.css

```
@@ -0,0 +1,469 @@
+.term__title {
+  margin-left: 8px;
+  font-size: 12.5px;
+  color: var(--ink-faint);
+}
+.term__body {
+  padding: 18px 18px 22px;
+  font-size: 13.5px;
+  line-height: 1.75;
+  color: var(--ink-dim);
+}
+.term__line { white-space: pre-wrap; word-break: break-word; }
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)